testsuite/integration-arquillian/servers/auth-server/common/fips/kc.java.security

```diff
@@ -40,11 +40,8 @@ fips.keystore.type=PKCS12
 # This is needed especially if we cannot add security provider "com.sun.net.ssl.internal.ssl.Provider BCFIPS" as a security provider.
 # OpenJDK has "SunX509" as default algorithm, but that one is not supported by BCJSSE. So adding the Sun provider delegating to BCFIPS is needed (as above)
 # or changing default algorithm as described here
-ssl.KeyManagerFactory.algorithm=PKIX
-fips.ssl.KeyManagerFactory.algorithm=PKIX
-
+ssl.KeyManagerFactory.algorithm=SunX509
```

Comment:
Minor: the value is not aligned with what is mentioned above in the comment. IMO it can be better to just comment those options?

But this is not a blocker for this PR IMO (I can change in the follow-up though...)

Reply:
You are right, I have modified `kc.java.security` to just contain the XMLDSig line. Let's see the tests...